microsoft/testsuites/core/hv_module.py

```diff
@@ -267,3 +268,45 @@ def _get_expected_modules(self, node: Node) -> List[str]:
                 modules.append(module)
 
         return modules
+
+    @TestCaseMetadata(
+        description="""
+        This testcase is to check if the hv_balloon driver has registered
+        correctly. It also checks if the hv_balloon driver is registered with
+        the right protocol version and parameters.
+
+        Steps,
+        1. Check dmesg if it contains hv-balloon driver registration string
+        2. Check dmesg if it contains hv-balloon protocol version number string
+        """,
+        priority=2,
+    )
+    def verify_hvballoon_for_reg(self, node: Node, log: Logger) -> None:
+        cat = node.tools[Cat]
+
+        drv_reg_str = "registering driver hv_balloon"
```

Comment:
currently, dynamic memory doesn't support on Azure, probably it is not mandatory on azure platform?